### DISCUSSION TITLE
Example server/client is slow even in the situation of local wired connection

### DISCUSSION THREAD

**Discussion by myl7** [aiortc/aioquic#270] [Q&A]
I modified the example server/client of aioquic, made it return `FileResponse`in `demo.py`, to test HTTP/3 (main logic is kept without modificatoin). The network is 100Mbps local wired connection but in the test the download speed can only reach about 700KB/s. The modified files can be found [here](https://github.com/myl7/xdptun/tree/main/test/h3). When downloading the CPU usage (both) is not high so CPU should not be the bottleneck. I am no sure whether it is expected or not but this is wiredly slow.

**Comment by devkdas** [comment]
Hello @myl7 

It seems to be you had a great knowledge about Scarlett and Web development with Python.(https://github.com/myl7/xdptun/tree/main/test/h3)

Could you please help these discussions

https://github.com/aiortc/aioquic/discussions/315
https://github.com/aiortc/aioquic/discussions/306

Looking forward for your reply.

Thank you.
